### PR TITLE
Add draft guide for integrating the event operator

### PR DESCRIPTION
**Pull request by SueChaplain**
New feature for 0.9.0. Draft guide for collaboration.

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
For issue https://github.com/kabanero-io/docs/issues/385

@mchenggit - please collaborate with me to get this complete for GA. I started working on it as a guide rather than Docs, but it doesn't quite work yet. The user has to be able to work through the guide with a local deployment and all the steps must work. There are still a number of issues to be solved:

- Are we capitalizing **E**vents **O**perator?
- diagram. Believe. listener 1.2.1, should say 1.0.1?
- Not all the example files are in the sample_crds directory. Maybe some are found in a Kabanero Foundation install instead. If so we need to call this out. I would have to put example files in a zip file for users to download from the online doc.

File: draft/integrating-events-operator/integrating-events-operator.md [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+//	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//	See the License for the specific language governing permissions and
+//	limitations under the License.
+//
+-->
+
+## What you'll learn
+
+In this guide, you will learn how to integrate the events operator to manage the flow of GitHub organization events through multiple pipelines. By setting up the webhook mediator, you will manage event flow based on the semantic versioning of your application stack projects.
+
```
Comment (mchenggit) [inline draft/integrating-events-operator/integrating-events-operator.md:43]: The main use cases are:
1) Support for Github organizational webhook, which only needs to be created once, versus having to create webhooks for every repository within the organization.
2) Support calling the version of Tekton pipeline that best matches the Appsody stack in the github repository, using semantic versioning.

File: draft/integrating-events-operator/integrating-events-operator.md [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+
+## Introduction
+
+The events operator allows users to define an event mediation flow for Kubernetes deployments. By using custom resource definitions (CRD),
+users can quickly construct mediation logic to receive, transform, and route JSON data structures. The transformation logic
+is based on the Common Expression Language (CEL).
+
+To integrate the events operator with Kabanero, you must use a function of the mediator framework, known as the webhook mediator, to process webhook events. The webhook mediator is used with a GitHub organization webhook to drive the pipelines.
+
```
Comment (mchenggit) [inline draft/integrating-events-operator/integrating-events-operator.md:57]: "You may use an event mediator to process webhook events. A mediator may be configured to drive versioned pipelines based on  Github organization webhooks events for typed repositories, such as Appsody repositories"

Comment (SueChaplain) [reply draft/integrating-events-operator/integrating-events-operator.md:57]: Hi @mchenggit - I think we need to simplify the language for the end user. Your suggestion has helped me understand a bit more, so I'm hoping that the following might be a good replacement for the original paragraph:

_To integrate the events operator with Kabanero, you use an event mediator to process events from a GitHub organization webhook. The event mediator is configured to filter and direct the flow of these events from different application stack repositories to the appropriate semantically-versioned pipeline. This process enables you to configure which events trigger which pipelines for making updates to different Kubernetes deployments._

What do you think?

Comment (mchenggit) [reply draft/integrating-events-operator/integrating-events-operator.md:57]: That is good. However, it's not the only integration. Though it is the only one we will demonstrate at the moment.

Comment (SueChaplain) [reply draft/integrating-events-operator/integrating-events-operator.md:57]: OK I'll add "you might use..." which implies an alternative, even if this does leave them wondering what that is ;-)

File: draft/integrating-events-operator/integrating-events-operator.md [hunk 3]
```diff
@@ -0,0 +1,486 @@
...
+2. Locates the event listener that best matches the stack, which is the event listener for stack version 0.3.3.
+3. Adds the pipeline parameters to the message body.
+4. Forwards the webhook message with the added parameters to the event listener.
+
+Note that in this example, the match for the Java OpenLiberty project (**project3**) is the event **listener 1.0.1**. However, no match exists for the Node.js project (**project1**).
+
+## Getting started
+
+When following this guide, use the configuration files in the [Events operator code repository](https://github.com/kabanero-io/events-operator/tree/prototype/sample_crds/example2). Ensure that you review and edit each `yaml` file when directed.
+
```
Comment (mchenggit) [inline draft/integrating-events-operator/integrating-events-operator.md:94]: Please remove the  repository, as the examples are now inline.

**Review by mchenggit — APPROVED** [review]

**Review by jantley-ibm — APPROVED** [review]
LGTM

**Pull request merged**